ted2zim/entrypoint.py

```diff
@@ -14,11 +14,29 @@ def main():
         prog=NAME, description="Scraper to create ZIM files from TED videos",
     )
 
+    parser.add_argument(
+        "--categories",
```

Comment:
As those are topics, can we use `--topics` instead?

Reply:
Yup. Renamed categories to topics